### ISSUE TITLE
Support GitLab-CI

### ISSUE THREAD

**Issue by dougthor42** [CleanCut/green#117]
Looks like the [GitLab-CI](https://about.gitlab.com/gitlab-ci/) is having the same issues that [Appveyor had with colors](http://help.appveyor.com/discussions/suggestions/197-support-ansi-color-codes).

If `green` is run within the gitlab-runner instance, then it should turn **off** Colorama so that colors are enabled.

I have no idea how you'd check for that, though... When the runner is installed on a Windows machine, it runs in a Windows terminal...

I guess the best way to support it would be to have a CLI option of "do not allow Colorama to convert color codes" or something like that.

**Comment by dougthor42** [comment]
Just FYI:
I'm working on a PR to add a "--nowindows" CLI option that will prevent Colorama from doing it's thing. This will allow people to get green working on GitLab just by passing an extra arg.

**Comment by CleanCut** [comment]
Sounds great!  Put it in the "Format Options" group next to --notermcolor

**Comment by dougthor42** [comment]
All done. Was a little touch-and-go with the unit tests for a second, but I think everything's good to go now.

**Comment by dougthor42** [comment]
For others who may find this:

After #118 has been merged and @CleanCut releases the next version of `green`, you'll need to use the `-t` and `-W` flags to get colors working if your `gitlab-runner` is on Windows:

```yaml
# .gitlab-ci.yml
job-name-here:
  script:
    - green -t -W test_file.py
```

Additional information at [gitlab-ci-multi-runner Issue #332](https://gitlab.com/gitlab-org/gitlab-ci-multi-runner/issues/332) and [MR #102](https://gitlab.com/gitlab-org/gitlab-ci-multi-runner/merge_requests/102).

**Comment by dougthor42** [comment]
By the way, this was fixed with PR #118. Sorry for forgetting to mark it!

**Comment by CleanCut** [comment]
No worries, thanks!
